best-practices/README.md

```diff
@@ -183,7 +183,9 @@ Email
 JavaScript
 ----------
 
-* Use CoffeeScript.
+* Use ES6 when possible with a tool like [6to5].
```

Comment:
How about using "Prefer", since we define it in the README?

* Prefer ES6 with a tool like 6to5.
* If 6to5 is not possible, use CoffeeScript (?)